python/ql/src/Functions/NonCls.ql

```diff
@@ -1,5 +1,5 @@
 /**
- * @name First parameter of a class method is not named 'cls'
+ * @name First argument of a class method is not named 'cls'
```

Comment:
`parameter` was the right word here, not `argument`. https://docs.python.org/3/faq/programming.html#faq-argument-vs-parameter

Reply:
This of course means we were using the _wrong_ term in `NonSelf.ql`...

Reply:
and we're also using the wrong term in the following 2 lines.

PEP8 also needs to be updated it seems, as it [says](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments):

> ### Function and Method Arguments
> Always use self for the first argument to instance methods.
>
> Always use cls for the first argument to class methods.